### PR TITLE
Bigtable AppendRow snippets

### PR DESCRIPTION
**Pull request by sangramql**

### CODE REVIEW

**Comment by sangramql** [comment]
@tseaver @sduskis @AVaksman
Please review the updated PR.

File: bigtable/docs/snippets_table.py [hunk 1]
```diff
@@ -387,5 +396,87 @@ def test_bigtable_table_row():
...
+    table = instance.table(TABLE_ID)
+    row = table.row(ROW_KEY2, append=True)
+
+    int_val = 3
+    row.increment_cell_value(COLUMN_FAMILY_ID, COL_NAME1, int_val)
+    # [END bigtable_row_increment_cell_value]
+    row.commit()
+
+    row_data = table.read_row(ROW_KEY2)
+    actual_value = row_data.cell_value(COLUMN_FAMILY_ID, COL_NAME1)
```
Comment (crwilcox) [inline bigtable/docs/snippets_table.py]: Similar to another PR, I would like to expand this name to be complete words and not include numbers.

**Comment by sangramql** [comment]
@sduskis can I still contribute to complete snippet PRs as open source dev?

**Comment by sduskis** [comment]
@sangramql, yes.

**Comment by crwilcox** [comment]
Ported to #7019

**Pull request closed without merging**